src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -736,17 +772,29 @@ public static View createBuildingView(ColonyView colony, BlockPos id, ByteBuf bu
             }
             catch (IndexOutOfBoundsException ex)
             {
-                Log.logger.error(
-                        String.format("A AbstractBuilding View (%s) has thrown an exception during deserializing, its state cannot be restored. Report this to the mod author",
-                                oclass.getName()), ex);
+                Log.logger.error(String
+                        .format("A AbstractBuilding View (%s) has thrown an exception during deserializing, its state cannot be restored. Report this to the mod author", oclass
+                                .getName()), ex);
                 view = null;
             }
         }
         else
         {
-            Log.logger.warn("Unknown AbstractBuilding type, missing View subclass, or missing constructor of proper format.");
+            Log.logger
+                    .warn("Unknown AbstractBuilding type, missing View subclass, or missing constructor of proper format.");
         }
 
         return view;
     }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)